tests/show_techsupport/test_techsupport.py

```diff
@@ -259,6 +259,26 @@ def config(request):
     """
     return request.getfixturevalue(request.param)
 
+@pytest.fixture
+def check_image_version(duthost):
+    """Skips this test if the SONiC image installed on DUT is older than 202112
+    Args:
+        duthost: Hostname of DUT.
+    Returns:
+        None.
+    """
+    skip_release(duthost, ["201811", "201911", "202012", "202106"])
```

Comment:
>201911 [](http://example.com/codeflow?start=38&length=6)

We plan to backport the feature [https://github.com/Azure/sonic-utilities/pull/1938](https://github.com/Azure/sonic-utilities/pull/1938) to 201911 and later branches. Could you not skip them?

Reply:
Maybe add these back after backport finish?
because if we make the UT run on those branch today, UT will fail and block some PRs.